stat/init.lua

```diff
@@ -137,7 +137,7 @@ local function stat(args)
 
     do
         local i = box.stat.net()
-        for k, v in pairs(i) do
+        for k, v in pairs({SENT = i.SENT, RECEIVED = i.RECEIVED}) do
```

Comment:
Tricky way :) As far as I understand Yaroslav proposed to do it in the most obvious way:

```lua
local box_stat_net = box.stat.net()
stats['stat.net.SENT.total'] = box_stat_net.SENT.total
stats['stat.net.SENT.rps'] = box_stat_net.SENT.rps
stats['stat.net.RECEIVED.total'] = box_stat_net.RECEIVED.total
stats['stat.net.RECEIVED.rps'] = box_stat_net.RECEIVED.rps
```